### PR TITLE
LRDOCS-9556 Microsoft Azure Blob Storage

### PR DESCRIPTION
**Pull request by evelyneduarte**
Ticket: https://issues.liferay.com/browse/LRDOCS-9556

### CODE REVIEW

**Comment by evelyneduarte** [comment]
@jrhoun Just checked the conflict, James pointed out that the difference is an article that got merged so it would be three lines in the rsts. Would you like us to fix them or is this not a problem?

File: docs/dxp/latest/en/system-administration/file-storage/other-file-store-types/microsoft-azure-blob-storage.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Microsoft Azure Blob Storage Service
+
+> Available for 7.4+ and 7.3 FP2+  
+
+Microsoft Azure Blob Storage is a cloud-based storage solution that DXP’s Azure Blob Storage Store uses to seamlessly store files to the cloud. Under the hood, it uses the Microsoft’s Blob Storage API, optimized to handle huge amounts of unstructured data.
+
+Before enabling this connection, you must have an active [Azure Blob Store account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&tabs=azure-portal), a configured [Connection String](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string), and an [Azure Blob Container](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-container-create?tabs=dotnet). Then, configure the Liferay instance to use Azure Blob Storage via *Control Panel*, and set `dl.store.impl` portal to use Azure Blob Storage as the default store.
```
Comment (jrhoun) [inline docs/dxp/latest/en/system-administration/file-storage/other-file-store-types/microsoft-azure-blob-storage.md:7]: > Then, configure the Liferay instance to use Azure Blob Storage via *Control Panel*, and set `dl.store.impl` portal to use Azure Blob Storage as the default store.

This sentence and paragraph is confusing to me. "Before enabling the connection I have to do x, y, and z. Then a, and b."?

How do I configure the Liferay instance to use Azure Blob Storage via the control panel?

How do I set `dl.store.impl`? What is "`dl.store.impl` portal"?

Comment (jrhoun) [reply docs/dxp/latest/en/system-administration/file-storage/other-file-store-types/microsoft-azure-blob-storage.md:7]: My _assumption_ for the intent being communicated here is this:

Before you enable the connection you have to have x, y, and z.

Once you have x, y, and z configured you must do the following:
* Configure the liferay instance via *Control Panel*
* Set the `dl.store.impl` portal property to use Azure Blob Storage as the default file store

**Comment by jrhoun** [comment]
Merged. Normally I'd want a little bit more work done on this but b/c this is high priority I went ahead and made some fixes/clarifications in the doc. See 004afba7c2ffc77d7a2c3097a08b52e78380ac08.

Also for future reference, please keep the ticket status' update to date. Thank you!

**Pull request closed without merging**